### PR TITLE
Add tslint to e2e tests

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?

- Adds ``` tslint ``` to e2e tests. 
- Fixes all tslint failures.
- Remove tsloader and add build of a project using ``` tsc  ```. 

Tests from tslint (witch will fix some minor issues as missing semicolon or whitespaces) and tsc build will be executed before running tests itself. 

The rules for tslint are taken from Dashboard part to keep it consistent. Anyway, there are five rules not known by tslint. It should be investigated and replaced for the correct ones (or removed). You can see all rules in ``` tslint.json ```.
```
    label-undefined
    no-constructor-vars
    no-duplicate-key
    no-trailing-comma
    no-unreachable
```

### What issues does this PR fix or reference?
https://github.com/redhat-developer/che-functional-tests/issues/486

### CODE REVIEW

**Comment by Katka92** [comment]
@Ohrimenko1988 Please could you take a look on the changes? Most of them are related to tslint fixes, but I would like you to take a look to ``` tslint.json ``` for rules (if you want add/change/remove some), ``` package.json ``` for new scripts and ``` mocha.opts ``` for changed configuration. Thank you!

**Comment by musienko-maxim** [comment]
Can one of the admins verify this PR?

**Comment by monaka** [comment]
@Katka92 Blocked by the ECA check. Please fix commit and force-push.

File: e2e/package.json [hunk 1]
```diff
@@ -18,6 +20,7 @@
     "mocha": "^6.1.4",
     "selenium-webdriver": "^3.6.0",
     "ts-node": "^8.0.3",
+    "tslint": "^5.16.0",
```
Comment (Ohrimenko1988) [inline e2e/package.json]: I have found that we have CQ for "5.10.0" tslint version
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16944

**Comment by Ohrimenko1988** [comment]
For me looks well.

File: e2e/package.json (hunk 1 above)
Comment (Katka92) [reply e2e/package.json]: Version changed to ```5.10.0```.

**Comment by Katka92** [comment]
ci-test

**Comment by Katka92** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-build

**Review by Ohrimenko1988 — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
ci-build

**Pull request merged**